src/Tests/dotnet-workload-install.Tests/GivenWorkloadManifestUpdater.cs

```diff
@@ -345,12 +339,56 @@ public void GivenWorkloadsAreOutOfDateUpdatesAreAdvertisedOnRestoringCommands(st
 
         }
 
+        [Fact]
+        public void TestSideBySideUpdateChecks()
+        {
+            var testDir = _testAssetsManager.CreateTestDirectory().Path;
+
+            (var updater1, var downloader1, var sentinalPath1) = GetTestUpdater(testDir: testDir, featureBand: "6.0.100");
+            (var updater2, var downloader2, var sentinalPath2) = GetTestUpdater(testDir: testDir, featureBand: "6.0.200");
+
+            updater1.BackgroundUpdateAdvertisingManifestsWhenRequiredAsync().Wait();
+            File.Exists(sentinalPath2).Should().BeFalse();
+
+            downloader1.DownloadCallParams.Should().BeEquivalentTo(GetExpectedDownloadedPackages("6.0.100"));
+
+            updater2.BackgroundUpdateAdvertisingManifestsWhenRequiredAsync().Wait();
+            File.Exists(sentinalPath2).Should().BeTrue();
+            downloader2.DownloadCallParams.Should().BeEquivalentTo(GetExpectedDownloadedPackages("6.0.200"));
+            var updateTime2 = DateTime.Now;
+
+            downloader1.DownloadCallParams.Clear();
+            updater1.BackgroundUpdateAdvertisingManifestsWhenRequiredAsync().Wait();
+            downloader1.DownloadCallParams.Should().BeEmpty();
+            File.GetLastAccessTime(sentinalPath1).Should().BeBefore(updateTime2);
+
+            downloader2.DownloadCallParams.Clear();
+            updater2.BackgroundUpdateAdvertisingManifestsWhenRequiredAsync().Wait();
+            var updateTime1 = DateTime.Now;
+            downloader2.DownloadCallParams.Should().BeEmpty();
+            File.GetLastAccessTime(sentinalPath2).Should().BeBefore(updateTime1);
+        }
+
+        private List<(PackageId, NuGetVersion, DirectoryPath?, PackageSourceLocation)> GetExpectedDownloadedPackages(string sdkFeatureBand)
```

Comment:
We can add a default value to this parameter, and then we don't have to pass it in for most of the call sites.

```suggestion
        private List<(PackageId, NuGetVersion, DirectoryPath?, PackageSourceLocation)> GetExpectedDownloadedPackages(string sdkFeatureBand = "6.0.100")
```